app/imports/components/Users/AdminPanel/UsersManagement/UsersList/setAdmin/setAdmin.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import { PropTypes } from 'prop-types';
+import { actions } from '/imports/api/users/actions.js';
+
+export default class SetAdmin extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.handleOpen = this.handleOpen.bind(this);
+        this.handleClose = this.handleClose.bind(this);
+        this.state = {
+            open: false,
+        };
+    }
+
+    handleOpen() {
+        this.setState({ open: true });
+    }
+
+    handleClose(choice) {
+        if (choice) { actions.manageAdmin(this.props.user); }
+        this.props.user.isAdmin = !this.props.user.isAdmin;
+
+        this.setState({ open: false });
+    }
+
+    render() {
+        const actiones = [
+            <FlatButton
+                label="No"
+                keyboardFocused
+                primary
```

Comment:
You shouldn't have two primary buttons. Add `secondary` prop to this one or next one.